docs/quickstart/svelte-kit.mdx

```diff
@@ -85,6 +85,10 @@ const posts = defineCollection({
     title: z.string(),
     summary: z.string(),
   }),
+	transform: (doc) => ({
+		...doc,
+		slug: doc.title.toLowerCase().replace(/ /g, '-'),
+	}),
 });
```

Comment:
The indentation does not look right: https://content-collections-git-fork-git-no-patch-1-nitilon.vercel.app/docs/quickstart/svelte-kit

Reply:
Completed.